faq.dd

```diff
@@ -393,15 +393,15 @@ $(ITEM printf, How do I get printf() to work with strings?)
 $(CCODE
 char s[8];
 strcpy(s, "foo");
-printf("string = '$(B %s)'\n", s);
+printf("string = '%s'\n", s);
```

Comment:
Custom escapes inside code are mightily difficult in latex, too. I'm very glad we're steering clear off them.